plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedView.java

```diff
@@ -77,6 +133,63 @@ protected String getStorageMetadataLocation(String materializedViewName)
         return table.getParameters().get(METADATA_LOCATION_PROP);
     }
 
+    @Test
+    public void testMaterializedViewCreatedFromPtfDoesNotActLikeLogicalView()
+    {
+        String viewName = "materialized_view_for_ptf_" + randomNameSuffix();
+        assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM TABLE(mock.system.random_function())");
+
```

Comment:
also test freshness as reported by `system.metadata.mvs`

Reply:
if only I knew there was a system table like this :D thank you!